lib/utils/unusedExpressionWatchers.js

```diff
@@ -10,20 +9,15 @@ const texts = require('./../texts');
 const unusedLeaves = new Set();
 /**
  * @description data about leaves code location
- * @type {Map<Object, {line: string, file: string}>}
+ * @type {Map<string>}
```

Comment:
This is invalid type for Map object, you should use Map<TKey, TValue> or Map.<TKEy, TValue>